notebooks/1.data_splits/nbconverted/1.data_splits.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Spliting Data
+# Here, we utilize the feature-selected profiles generated in the preceding module notebook [here](../0.freature_selection/), focusing on dividing the data into training, testing, and holdout sets for machine learning training.
```

Comment:
```suggestion
# Here, we utilize the feature-selected profiles generated in the preceding module notebook [here](../0.feature_selection/), focusing on dividing the data into training, testing, and holdout sets for machine learning training.
```

Once you fix the typo

Reply:
Also, I think I reviewed this notebook already? Maybe you forgot a github merge along the way?